src/js/views/patients/shared/list_views.js

```diff
@@ -24,13 +25,16 @@ const MaximumCountNarrowedTemplate = hbs`
 `;
 
 const CountView = View.extend({
+  initialize() {
+    const listType = this.getOption('isFlowList') ? 'flows' : 'actions';
```

Comment:
moved this into the view so we only have to pass in `isFlowList` and the `get(...` logic is only needed here.